app/src/renderer/vuex/modules/delegates.js

```diff
@@ -69,28 +70,40 @@ export default ({ node }) => {
     },
     async getDelegates({ commit, dispatch }) {
       commit(`setDelegateLoading`, true)
-      let validators = await node.getCandidates()
-      let { validators: validatorSet } = await node.getValidatorSet()
-      for (let validator of validators) {
-        validator.isValidator = false
-        if (validatorSet.find(v => v.pub_key === validator.pub_key)) {
-          validator.isValidator = true
+      try {
+        let validators = await node.getCandidates()
+        let { validators: validatorSet } = await node.getValidatorSet()
+        state.error = null
```

Comment:
why is this necessary?

Reply:
this resets the error on a successful load